src/sass/patterns/_containers.scss

```diff
@@ -0,0 +1,16 @@
+// Containers
+// ==========
+
+// Restrict fixed/absolute containers to half the viewport
+// ie. used on toast containers to unblock header links
```

Comment:
How do I trigger a toast?

Reply:
I think the simplest is to stop your server/container and the click the "Sync GitHub Repositories" button. (But again, functionally the toast is the same as on MetaDeploy.)